OCKSample/Main/Care/CareViewController.swift

```diff
@@ -299,12 +318,44 @@ class CareViewController: OCKDailyPageViewController {
         var query = OCKTaskQuery(for: date)
         query.excludesTasksWithNoEvents = true
         do {
-            return try await storeManager.store.fetchAnyTasks(query: query)
+            let tasks = try await storeManager.store.fetchAnyTasks(query: query)
+            // Remove onboarding tasks from array
+            return tasks.filter { $0.id != TaskID.onboarding }
         } catch {
             Logger.feed.error("\(error.localizedDescription, privacy: .public)")
             return []
         }
     }
+
+    @MainActor
+    private func checkIfOnboardingIsComplete() async -> Bool {
+        var query = OCKOutcomeQuery()
+        query.taskIDs = [TaskID.onboarding]
+
+        guard let store = AppDelegateKey.defaultValue?.store else {
```

Comment:
I modified this method and it's different from what we did in class. You should replace your `checkIfOnboardingIsComplete()` with the one here